index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+        body {
+            background-image: url("Project materials/cover.png");
+            height: 100%;
+            width: 100%;
+            background-position: center;
+            background-repeat: no-repeat;
+            background-size: cover;
+            position: relative;
+        }
+
+        body::before {
+            content: '';
+            display: block;
+            width: 100%;
+            height: 100%;
+            position: absolute;
+            top: 0;
+            left: 0;
+            background-color: rgba(0, 0, 0, 0.5);
+            z-index: -1;
+        }
+
+        .container {
+            display: flex;
+            max-width: 1050px;
+            padding: 10%;
+            flex-direction: column;
+            align-items: center;
+            gap: 40px;
+            margin: auto;
+        }
+
+        h1 {
+            color: white;
+            text-align: center;
+            display: block;
+            font-family:'Times New Roman', Times, serif;
+            font-size: 5rem;
+            font-style: normal;
+            font-weight: 50rem;
+            line-height: 80px;
+            letter-spacing: 0.2px;
+            width: 573px;
+            height: 143px;
+        }
+
+        h4 {
```

Comment:
I suggest you to use complex selector for h1 and h4: `h1, h4 {...}` for common styles and separately add only personal styles. 
And again, better use classes instead of tag names